specification/common/transactions/assets/RegisterDelegateAsset.yaml

```
@@ -7,14 +7,11 @@ properties:
   delegate:
     type: object
     properties:
```

Please add the `required` prop at this level

```suggestion
    required:
      - address
      - username
      - publicKey
    properties:
```

And fix the `required` root prop.

```yaml
required:
  - delegate
```